attendance/API_HR/c-hr.php

```diff
@@ -2849,9 +2851,24 @@ public static function getDisabledUsersList() {
                 $newRows[] = $pp;
             }
         }
+        
+        return $newRows;
```

Comment:
As i had mentioned FE will depend on BE response, your response must have total pages, current page, next page, previous page